packages/ui-components/src/components/relative-time-picker/relative-time-picker.config.ts

```diff
@@ -0,0 +1,429 @@
+import { ERelativeTimeComparisonConfig, IRelativeTimePickerOption } from './relative-time-picker.types';
+
+export const TIME_RANGE_UPDATE_INTERVAL = 10000;
+export const CUSTOMIZE_INTERVAL_KEY = 'customize-interval';
+export const CUSTOMIZE_INTERVAL_LABEL = 'Customize Interval';
+export const DROPDOWN_MAX_HEIGHT = '290px';
```

Comment:
CSS on the JS? 😬

Reply:
Hehe it was a prop from the dropdown that was available to set this height but I can do it with in the scss file 👍